cli/main.rs

```diff
@@ -51,11 +51,17 @@ fn main() {
 }
 
 fn repl(mut jstime: jstime::JSTime) {
+    use dirs::home_dir;
     use rustyline::{error::ReadlineError, Editor};
 
+    let mut history_path = home_dir().unwrap();
```

Comment:
should've noticed this earlier, but if `home_dir()` fails (idk, maybe someone is running jstime on a gameboy advance) this will also crash. not sure it matters too much :sweat_smile: